meinberlin/assets/scss/components/_poll.scss

```diff
@@ -311,3 +311,8 @@ $checkbox-size: 20px;
         right: 4*$spacer;
     }
 }
+
+.poll-slider__item--highlight {
+    padding: $spacer;
+    background-color: $brand-primary-tint;
+}
```

Comment:
cool that it's working but the design was changed due to a11y, can be seen here: https://app.zeplin.io/project/5b051eef7aaa417d2227c55c/screen/5f466bb2f5b0f43658eac1a3